installing/installing_openstack/installing-openstack-installer.adoc

```diff
@@ -0,0 +1,57 @@
+[id="installing-openstack-installer"]
+= Installing a cluster on OpenStack
+include::modules/common-attributes.adoc[]
+:context: installing-openstack-installer
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+OpenStack.
+
+.Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+
+* Have access to an OpenStack administrator's account
+
+* In order to run the {product-title} installer in OpenStack, your quota should meet these requirements:
++
+.Minimum recommended resources for a default OpenShift cluster on OpenStack
+[options="header"]
+|================================
+|Resource              | Value
+|Floating IP addresses | 2
+|Routers               | 1
+|Subnets               | 1
+|RAM                   | 112 GB
+|vCPUs                 | 28
+|Volume storage        | 175 GB
+|Instances             | 7
+|Security groups       | 3
+|Security group rules  | 60
+|================================
```

Comment:
Same deal as with `installing-openstack-installer-custom.adoc` -- we should add the Swift-related quotas here.

Reply:
I modularized this so that it's the same document appearing in both places. Change should be apparent in my EOD update to this PR. Thanks!